plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/finish/index.tsx

```diff
@@ -14,12 +14,22 @@ import './style.scss';
 import { recordPaymentsOnboardingEvent } from '~/settings-payments/utils';
 
 export const FinishStep: React.FC = () => {
-	const { context, closeModal, sessionEntryPoint } = useOnboardingContext();
+	const { context, currentStep, closeModal, sessionEntryPoint } =
+		useOnboardingContext();
 
 	return (
 		<>
 			<WooPaymentsStepHeader onClose={ closeModal } />
 			<div className="settings-payments-onboarding-modal__step--content">
+				{ currentStep?.errors && currentStep.errors.length > 0 && (
```

Comment:
If there are unhandled errors sent by the backend, always show them. It is better to inform the user and ourselves).